### PR TITLE
Feat/ab test service update

### PR DESCRIPTION
**Pull request by mvhou**
Updated the localStorage things to work with the abStorageService

### CODE REVIEW

File: frontend/public/widget/co2okWidgetMark-2786c947.js [hunk 1]
```diff
@@ -525,7 +526,7 @@ let Co2okWidget = {
 		// Manual AB-switch
 		var urlParams = new URLSearchParams(window.location.search);
 		var co2ok_AB_param = urlParams.get('co2ok_ab');
-		let co2ok_AB_test = JSON.parse(localStorage.getItem('co2ok_ab_hide'));
+		let co2ok_AB_test = localStorage.getItem('co2okAB');
 
 		// Vela uses WC, which still uses cookies to store the AB state
```
Comment (Mil0dV) [inline frontend/public/widget/co2okWidgetMark-2786c947.js:531]: Not anymore. Although changing this should prob be another PR in an enterprise organisation, let's do it in this one for now.

File: frontend/public/widget/co2okWidgetMark-587cf528.js [hunk 2]
```diff
@@ -642,14 +642,14 @@ Co2okWidget.manualABSwitch()
...
+    Co2okWidget.loadResources()
     return
   }
 })
-
 */
 
 
+
 // Comment this out, or remove it entirely, if AB testing is turned on again
```
Comment (Mil0dV) [inline frontend/public/widget/co2okWidgetMark-587cf528.js:653]: I don't understand why this is necessary. Shouldn't manualABSwitch() return true if the AB test is disabled (and thus always run both these functions)?

**Review by Mil0dV — CHANGES_REQUESTED** [review]

File: frontend/public/widget/co2okWidgetMark-587cf528.js (hunk 2 above)
Comment (mvhou) [reply frontend/public/widget/co2okWidgetMark-587cf528.js:653]: No, if the AB test is disabled manualABSwitch() never returns anything. The value of co2okAB (or co2ok_ab_hide) never gets set in localStorage, so the first condition (co2ok_AB_test === null) will always be true.
